happypanda/app_constants.py

```diff
@@ -15,38 +15,38 @@
 import os, sys, enum
 import qtawesome as qta
 
-try:
-    import settings
-    from database import db_constants
-except ImportError:
-    from . import settings
-    from .database import db_constants
+from happypanda import settings
+from happypanda.database.arguments import args
+from happypanda.database import db_constants
 
 # Version number
 vs  = '1.0'
 DEBUG = False
 
-OS_NAME = ''
 if sys.platform.startswith('darwin'):
-	OS_NAME = "darwin"
+    OS_NAME = "darwin"
 elif os.name == 'nt':
-	OS_NAME = "windows"
+    OS_NAME = "windows"
+    import ctypes
```

Comment:
Yeah I do plan to move it though along with all the others sprinkled throughout the codebase, heck we even have imports that do nothing.